scripts/envoy-boringssl-docker.sh

```diff
@@ -28,4 +28,4 @@ if [[ ! -f "${BASE_DIR}"/cert.pem || ! -f "${BASE_DIR}"/key.pem ]]; then
   openssl req -x509 -new -batch -nodes -subj '/CN=localhost' -keyout "${BASE_DIR}"/key.pem -out "${BASE_DIR}"/cert.pem
 fi
 
-docker run --rm -ti -p 9000:9000 $CPUSET_PARAM --security-opt seccomp=unconfined --security-opt apparmor=unconfined $DEVS_PARAM --cap-add=SYS_ADMIN --cap-add=IPC_LOCK -v "${BASE_DIR}"/cert.pem:/etc/envoy/tls/tls.crt -v "${BASE_DIR}"/key.pem:/etc/envoy/tls/tls.key -v $CONFIG_FILE_PARAM:/etc/envoy/config/envoy-conf.yaml envoy-boringssl-qat:devel --cpuset-threads
+docker run --rm -ti -p 9000:9000 $CPUSET_PARAM --security-opt seccomp=unconfined --security-opt apparmor=unconfined $DEVS_PARAM --cap-add=SYS_ADMIN --cap-add=IPC_LOCK -v "${BASE_DIR}"/cert.pem:/etc/envoy/tls/tls.crt -v "${BASE_DIR}"/key.pem:/etc/envoy/tls/tls.key -v $CONFIG_FILE_PARAM:/etc/envoy/config/envoy-conf.yaml -v /dev/hugepages/qat:/dev/hugepages/qat envoy-boringssl-qat:devel --cpuset-threads
```

Comment:
Does QAT need hugepages for crypto ops?

Reply:
Turns out QAT 4.10 tries to create a temporary file in `/dev/hugepages/qat/` during section activation and that fails if the directory isn't there. I'm not quite sure if this is intended behavior and I'm also not sure if the huge page is actually used during crypto operations. At least QAT provider isn't directly using the huge page allocation functions -- the QAT continuous memory allocation functions may use them in the background.

Reply:
I see. I suppose QAT touches hugepages if DC instances are enabled in the config file.